src/Identity/v3/Params.php

```diff
@@ -195,6 +195,15 @@ public function userIdQuery(): array
         ];
     }
 
+    public function userIdQueryUnderscore(): array
```

Comment:
Why creating new function? 
As far as I see https://docs.openstack.org/api-ref/identity/v3/#credentials 
`POST /v3/credentials` and `GET /v3/credentials` are using the same parameter type `user_id`.